cnxepub/tests/test_formatters.py

```diff
@@ -292,6 +292,7 @@ def test_document_auto_generate_ids(self):
         from ..formatters import HTMLFormatter
 
         random.seed(1)
+        n = random.randint(0, 100000)
```

Comment:
This shouldn't be necessary.  If the test isn't passing without this, we need to investigate into it.